cmd/web5/main.go

```diff
@@ -7,9 +7,10 @@ import (
 )
 
 type CLI struct {
-	DID didCmd `cmd:"" help:"Interface with DID's."`
-	DIDJWK didJWKCmd `cmd:"" name:"did:jwk" help:"Manage did:jwk's."`
-	DIDWeb didWebCmd `cmd:"" name:"did:web" help:"Manage did:web's."`
+	DID struct {
```

Comment:
Nested struct 👀 

(did it because I didn't love the idea of having a file `cmd_did.go` with this single struct in it)